### PR TITLE
[zh-cn]: update the translation of Number.POSITIVE_INFINITY/NEGATIVE_INFINITY

### PR DESCRIPTION
**Pull request by JasonLamv-t**
…GATIVE_INFINITY

### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None. 

### Related issues and pull requests

#15736

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/number/negative_infinity/index.md [hunk 1]
```diff
@@ -3,41 +3,47 @@ title: Number.NEGATIVE_INFINITY
...
-- 0 乘以 `NEGATIVE_INFINITY` 为 `NaN`.
-- NaN 乘以 `NEGATIVE_INFINITY` 为 `NaN`.
-- `NEGATIVE_INFINITY` 除以任何负值（除了 `NEGATIVE_INFINITY）`为 `POSITIVE_INFINITY`。
-- `NEGATIVE_INFINITY` 除以任何正值（除了 `POSITIVE_INFINITY`）为 `NEGATIVE_INFINITY`。
-- `NEGATIVE_INFINITY` 除以 `NEGATIVE_INFINITY` 或 `POSITIVE_INFINITY` 是 `NaN`。
-- 任何数除以 `NEGATIVE_INFINITY` 为 0。
+你可以使用 `Number.NEGATIVE_INFINITY` 属性来表示成功时返回有限数值的错误条件。不过，{{jsxref("NaN")}} 更适合于这种情况。
 
-为了成功返回一个有限值，你可能会使用 `Number.NEGATIVE_INFINITY` 属性来判断是否显示一个条件错误。然而 {{jsxref("Global_Objects/isFinite", "isFinite")}} 方法更适合这种情况。
+因为 `NEGATIVE_INFINITY` 是 {{jsxref("Number")}} 的静态属性，所以你应该始终使用 `Number.NEGATIVE_INFINITY`，而不是一个数字值的属性。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/negative_infinity/index.md:35]: Could this be aligned with https://github.com/mdn/translated-content/pull/15958/files#diff-b0639b38f99e63839b17bd05159ae23e5d9141f846b6b37e5d9c3c7071e53609R22

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/number/negative_infinity/index.md:35]: No problem.

File: files/zh-cn/web/javascript/reference/global_objects/number/positive_infinity/index.md [hunk 2]
```diff
@@ -3,40 +3,47 @@ title: Number.POSITIVE_INFINITY
...
+- 任何正值，包括 `POSITIVE_INFINITY`，乘以 `POSITIVE_INFINITY` 等于 `POSITIVE_INFINITY`。
+- 任何负值，包括 {{jsxref("Number.NEGATIVE_INFINITY", "NEGATIVE_INFINITY")}}，除以 `POSITIVE_INFINITY` 等于 {{jsxref("Number.NEGATIVE_INFINITY", "NEGATIVE_INFINITY")}}。
+- 任何正数除以 `POSITIVE_INFINITY` 都是[正零](https://zh.wikipedia.org/wiki/0)（正如 [IEEE 754](https://zh.wikipedia.org/wiki/IEEE_754) 中所定义的）。
+- 任何负数除以 `POSITIVE_INFINITY` 都是[负零](https://zh.wikipedia.org/wiki/−0)（正如 [IEEE 754](https://zh.wikipedia.org/wiki/IEEE_754) 中所定义的）。
+- 零乘以 `POSITIVE_INFINITY` 等于 {{jsxref("NaN")}}。
+- {{jsxref("Global_Objects/NaN", "NaN")}} 乘以 `POSITIVE_INFINITY` 等于 {{jsxref("NaN")}}。
+- `POSITIVE_INFINITY`，除以任何负值（除了 {{jsxref("Number.NEGATIVE_INFINITY", "NEGATIVE_INFINITY")}}）都等于 {{jsxref("Number.NEGATIVE_INFINITY", "NEGATIVE_INFINITY")}}。
+- `POSITIVE_INFINITY`，除以任何正值（除了 `POSITIVE_INFINITY`）都等于 `POSITIVE_INFINITY`。
+- `POSITIVE_INFINITY`，除以 {{jsxref("Number.NEGATIVE_INFINITY", "NEGATIVE_INFINITY")}} 或 `POSITIVE_INFINITY`，都等于 {{jsxref("NaN")}}。
+- `Number.POSITIVE_INFINITY < x` 对于任何不是 `POSITIVE_INFINITY` 的数字 _x_ 都为真。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/number/positive_infinity/index.md:31]:
```suggestion
- `Number.POSITIVE_INFINITY > x` 对于任何不是 `POSITIVE_INFINITY` 的数字 _x_ 都为真。
```

**Review by yin1999 — COMMENTED** [review]
One nit here

**Review by yin1999 — APPROVED** [review]
Thank you @JasonLamv-t :+1:

**Pull request merged**